website/lib/changelog/generate-component-changelog-entries.mjs

```diff
@@ -153,9 +100,9 @@ const updateComponentVersionHistory = (componentChangelogEntries, version) => {
       .map((entry) => {
         // If the component is a form primitive, we want to keep the component name in the description
         if (
-          allComponentsPath[componentName] ===
+          allComponentPaths[componentName] ===
```

Comment:
[issue/blocker] maybe I'm missing something, but `allComponentPaths` is not anymore an object, but an array, so `allComponentPaths[componentName]` returns `undefined`